docs/manual/manual_stock_led.rst

```diff
@@ -0,0 +1,314 @@
+until the value is changed and saved again. So, if you restart your module, you will find that it now starts with the same color and intensity as configured 
+through the Control Center.
+
+An important note is that it is not required to set ``RGB LED Update Color`` in order to have your module boot with the previously saved color. The module will 
+always boot with the value loaded from persistent memory. Suppose you want your module to default to magenta. We can follow the same steps as above with 
+IQ Control Center to achieve this, but this example will use the Python API. Note that we do not call ``module.set("rgb_led", "update_color")``, but the 
+module reboots with the value set and saved by the program.
+
+.. code-block:: python
+
+    import iqmotion as iq
+
+    # Module Communication - This must be updated with the correct parameters for your module's serial communication
+    com = iq.SerialCommunicator("COM3", baudrate=115200)
+
+    # Using a Vertiq 40-06 Gen 2 module with speed firmware
+    module = iq.Vertiq4006(com, firmware="speed")
+
+    #Set and save the RGB LED to magenta (red and blue only)
+    module.set("rgb_led", "red", 100)
+    module.save("rgb_led", "red")
+    module.set("rgb_led", "green", 0)
+    module.save("rgb_led", "green")
+    module.set("rgb_led", "blue", 255)
+    module.save("rgb_led", "blue")
+
+    #Reboot the module to show that it will boot to the color we just set
+    module.set("system_control", "reboot_program")
+
+White LED Configuration
+==========================
+Your module has control over the two series white LEDs' intensity as well as their strobing output. Strobing configuration is identical for both the RGB and white LEDs, 
+and is covered in :ref:`strobing_configuration`. By default, the white LED intensity is 0% and strobing is enabled. White LED control parameters are 
+found in the :ref:`white LED message table <white_led_message_table>`.  The LED intensity can be an integer percentage [0, 100]% where 100% indicates the maximum intensity. Please note that the 
+two white LEDs are controlled in series, and cannot be controlled individually.
+
+.. warning::
+    The white LEDs are extremely bright. Please do not look directly into the white LEDs while they are in use. Additionally, while operating the LED board at high 
+    intensities, the LED board may get very hot. Please take care when handling your LED board to avoid injury.
+
+Since its default intensity is 0%, the white LEDs are off by default. Suppose you would like to turn on the LEDs with minimum intensity (all examples will use low 
+intensities in order to avoid intense light while benchtop testing). To do so, we can use the :ref:`Python API <getting_started_python_api>`.
+
+Once the intensity is configured above 0%, you will see the white LED flash with its default :ref:`strobing pattern <strobing_configuration>` in which it blinks on 
+and off 3 times, remains off, then restarts the pattern.
```

Comment:
You have a nice video of this default pattern above, maybe worth linking to it here for people who may have skipped it before